packages/forms-web-app/src/controllers/selected-appeal/questionnaire-details/constraints-details-rows.js

```diff
@@ -94,4 +95,14 @@ exports.constraintsRows = (caseData) => {
 			condition: () => caseData.uploadDefinitiveMapStatement
 		}
 	];
+
+	if (user === 'lpa-user') {
```

Comment:
Minor point but you might want to import the LPA_USER_ROLE constant from common package rather than using the string here, for an example see packages/forms-web-app/src/controllers/selected-appeal/appeal-details/index.js